src/tests/ascii/CMakeLists.txt

```diff
@@ -1,9 +1,4 @@
-include_directories(${PROJECT_BINARY_DIR}/src)
-
-project(ascii)
-
 add_executable(test_ascii test_ascii.f90)
 target_link_libraries(test_ascii fortran_stdlib)
 
-add_test(test_ascii ${PROJECT_BINARY_DIR}/test_ascii)
-
+add_test(NAME ASCII COMMAND $<TARGET_FILE:test_ascii>)
```

Comment:
👍

Reply:
old CMake syntax is painful in every way, yes